src/test/ui/consts/const_let_assign3.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(keyword_generics)]
+
```

Comment:
Does this test use the feature? I don't think so

Reply:
everything with a `const fn` uses it, and this test actually caught a bug. I just sprinkled the feature on random tests in `src/test/ui/consts` and fixed bugs as they appeared.